integration-tests/opentelemetry.spec.js

```diff
@@ -121,15 +121,15 @@ describe('opentelemetry', () => {
         OTEL_LOG_LEVEL: 'debug',
         DD_TRACE_SAMPLE_RATE: '0.5',
         OTEL_TRACES_SAMPLER: 'traceidratio',
-        OTEL_TRACES_SAMPLER_ARG: '0.1',
+        OTEL_TRACES_SAMPLER_ARG: '1.0',
         DD_TRACE_ENABLED: 'true',
         OTEL_TRACES_EXPORTER: 'none',
         DD_RUNTIME_METRICS_ENABLED: 'true',
         OTEL_METRICS_EXPORTER: 'none',
         DD_TAGS: 'foo:bar,baz:qux',
-        OTEL_RESOURCE_ATTRIBUTES: 'foo=bar1,baz=qux1',
-        DD_TRACE_PROPAGATION_STYLE: 'datadog',
-        OTEL_PROPAGATORS: 'datadog,tracecontext',
+        OTEL_RESOURCE_ATTRIBUTES: 'foo+bar13baz+qux1',
```

Comment:
If I'm understanding this correctly, we're no longer considering this as "invalid" because we don't have a set of recognized values, so that's why the `otelInvalid.length` went down?

Reply:
Yes precisely, I think that should also be the expected behavior cross tracers.